Lib/Emscripten/Emscripten.cpp

```diff
@@ -413,6 +413,64 @@ DEFINE_INTRINSIC_FUNCTION(env, "abort", void, emscripten_abort, I32 code)
 	throwException(Runtime::Exception::calledAbortType);
 }
 
+DEFINE_INTRINSIC_FUNCTION(env, "nullFunc_i", void, emscripten_nullFunc_i, I32 code)
+{
+	throwException(Runtime::Exception::calledUnimplementedIntrinsicType);
```

Comment:
I'm not sure if this should be `calledUnimplementedIntrinsicType ` or `calledAbortType`... Both seem like a reasonable choices to me.

Reply:
I think it should be `uninitializedTableElementType`, but that expects a table reference and index, and I'm not sure it's practical to get the index from Emscripten. I'd say `calledAbortType` for now.

Reply:
Updated, thanks!